doc/source/User_guide/extensions.rst

```diff
@@ -112,6 +140,21 @@ They are small automated workflow with a simple UI.
             Import a CSV file containing sources layout and power dissipation information.
 
 
+Circuit extensions
+~~~~~~~~~~~~~~~~~~
+
+Pre-installed extensions are available at Circuit level.
+They are small automated workflow with a simple UI.
```

Comment:
```suggestion
They are small automated workflows with a simple UI.
```

Reply:
I was going to ask for the same change, why is this solved but no changes performed ?